docs/docs/external/getting-started/test-network/setup-local-docker.md

```diff
@@ -133,6 +133,10 @@ Follow the instructions below to build and launch the networks:
   ```bash
   make start-interop-local CHAINCODE_NAME=<chaincode-name>
   ```
+- _To launch the networks with 2 orgs, run_:
```

Comment:
"2 orgs, each with a peer,". Add in parantheses something like "(this will enable more variation and experimentation, which you can attempt after testing interoperation protocols across basic network configurations)".

Do this in all the Setup files.